HOWTO-make-a-release.md

```diff
@@ -142,6 +133,19 @@ You will need to checkout at least three working trees:
         (cd openssl;
          git worktree add ../openssl-1.1.1 OpenSSL_1_1_1-stable)
 
+## Freeze the source repository
+
+The day before the release, freeze the main repository.  This locks out
```

Comment:
I think we're now suggesting three business days before the release

Reply:
Fixed